coding-concepts.js

```diff
@@ -8,44 +8,45 @@
 // --------------------1) What will this log?
 
 var cohort = "Delta 2021"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10
+// b) Verify and explain: I counted all spaces including blank spaces and it was 10.
 
 
 // --------------------2) What will this log?
 
 var greeting = "Hello World!"
-// console.log(greeting[3])
+console.log(greeting[3])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 'l'
+// b) Verify and explain: it's the 3rd index of a zero indexed system
 
 
 // --------------------3) What will this log?
 
 var languages = ["JavaScript", "Ruby", "Python", "C++"]
 var index = 1
-// console.log(languages[index])
+console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 'Ruby'
+// b) Verify and explain: index is equal to 1 and in a zero indexed file that will the 2nd item
 
 
 // --------------------4) What will this log?
 
 var weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "SATURDAY", "SUNDAY"
+// b) Verify and explain:toUpperCase() is a built in function that changes lowercase letters to all uppercase.
+// Correct answer was that toUpperCase is not a function. Trick question got me!
```

Comment:
Glad it worked! Yeah, keeping track of the data type string vs array and knowing which methods work on each is part of the job.